package.json

```diff
@@ -35,6 +35,7 @@
     "react-intl": "^2.1.2",
     "react-router": "^2.4.1",
     "react-treeview": "^0.4.5",
+    "remove-accents": "^0.4.0",
```

Comment:
Indeed, [`andrewrk/node-diacritics`](https://github.com/andrewrk/node-diacritics) has [more support](https://github.com/andrewrk/node-diacritics/stargazers), supports [Unicode private range](https://github.com/andrewrk/node-diacritics/blob/master/index.js#L310), and seems more resilient to encoding support, since the charmap is [escaped](https://github.com/andrewrk/node-diacritics/blob/master/index.js#L297). It is also probably more performant, since it doesn't do dynamic RegExes.

I don't think we should do it without a library, though. The charmap won't change much, but there are pesky little issues around it.